src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -499,9 +526,13 @@ void manualRegistrationToolBox::displayButtons(bool show)
     }
 }
 
-
 void manualRegistrationToolBox::updateLabels(int left,int right)
 {
-    d->numberOfLdInLeftContainer->setText( "Number of landmarks in left container  : " + QString::number(left));
-    d->numberOfLdInRightContainer->setText("Number of landmarks in right container : " + QString::number(right));
+    d->numberOfLdInLeftContainer->setText( "Number of landmarks in left container: " + QString::number(left));
+    d->numberOfLdInRightContainer->setText("Number of landmarks in right container: " + QString::number(right));
+
+    if((left>0) && (left==right))
+    {
+        disableComputeButton(false);
+    }
```

Comment:
well, if left=right=0, a reset just have been runned, or it's a new data, so it would be a duplication :)